events/janus_wsevh.c

```diff
@@ -150,9 +159,12 @@ static const char *protocol = NULL, *address = NULL;
 static char path[256];
 static int port = 0;
 static struct lws_context *context = NULL;
+#if ((LWS_LIBRARY_VERSION_MAJOR == 3 && LWS_LIBRARY_VERSION_MINOR >= 2) || LWS_LIBRARY_VERSION_MAJOR >= 4)
+	static lws_sorted_usec_list_t sul_stagger = { 0 };
```

Comment:
fix indentation ;-)

Reply:
k got it now :]
i always intended the code when i was mainly developing c++ but thats definetly user perspective... will address the others as well..